src/yoroi-wallets/walletManager.ts

```diff
@@ -522,9 +518,8 @@ class WalletManager {
     await this.closeWallet()
   }
 
-  async removeCurrentWallet() {
+  async removeCurrentWallet(walletId: YoroiWallet['id']) {
```

Comment:
I've updated our docs, walletId: string -> YoroiWallet['id']
```suggestion
  async removeWallet(walletId: YoroiWallet['id']) {
```